src/Pages/LoginPage/LoginPage.js

```diff
@@ -0,0 +1,30 @@
+import LoginForm from "Components/LoginForm/LoginForm";
+import { Navigate } from "react-router";
+import useAuthentication from "Services/firebase/useAuthentication";
+import useTitle from "Utils/useTitle";
+
+const LoginPage = function () {
+  useTitle("Bragi 3000 - Login");
+
+  const { authReady, loggedIn } = useAuthentication();
+
+  if (!authReady)
```

Comment:
Duplicated code from SignupPage. However for this small amount of code I don't think we need to extract it.

Reply:
Yeah, this usage (the page should only be accessible when you're *not* logged in) is quite rare. I first wanted to create some `PrivateRoute` component (for routes only accessible when you *are* logged in), but even that will only be used in the `App` page (I think), so in the end I decided to do it like this, with the `useAuthentication` hook abstracting away some of the logic.